### PR TITLE
This change makes S3Proxy able to validate presigned url which 

### PR DESCRIPTION
**Pull request by shenghu**
has override parameters, i.e. "content-disposition",
"response-content-encoding".

### CODE REVIEW

**Comment by shenghu** [comment]
@andrewgaul can you help check why Travis failed? Thanks.

**Comment by andrewgaul** [comment]
@shenghu Something has changed about how Travis configures Python -- I have not figured out a solution yet.  These failures are unrelated to your code.

File: src/test/java/org/gaul/s3proxy/AwsSdkTest.java [hunk 1]
```diff
@@ -300,6 +340,40 @@ public void testAwsV2UrlSigning() throws Exception {
...
+
+        ResponseHeaderOverrides headerOverride = new ResponseHeaderOverrides();
+
+        headerOverride.setContentDisposition("attachment; " + blobName);
+        headerOverride.setContentType("text/plain");
+        generatePresignedUrlRequest.setResponseHeaders(headerOverride);
+
+        Date expiration = new Date(System.currentTimeMillis() +
+                TimeUnit.HOURS.toMillis(1));
+        generatePresignedUrlRequest.setExpiration(expiration);
```
Comment (andrewgaul) [inline src/test/java/org/gaul/s3proxy/AwsSdkTest.java]: How does this differ from `testAwsV2SignatureWithOverrideParameters`?  You set an expiration here but do not check the overridden content disposition and type.  Should you fold this test into the first one?

Comment (shenghu) [inline src/test/java/org/gaul/s3proxy/AwsSdkTest.java]: testAwsV2SignatureWithOverrideParameters is testing w/ non-presigned URL. testAwsV2UrlSigningWithOverrideParameters is testing w/ presigned URL, which is the same as testAwsV2UrlSigning(). I can add verification for expiration, content dispositioin and type. Or are you insisting to merge this test case to testAwsV2SignatureWithOverrideParameters?

Comment (andrewgaul) [inline src/test/java/org/gaul/s3proxy/AwsSdkTest.java]: Sorry I misread this -- you are correct.

Comment (andrewgaul) [inline src/test/java/org/gaul/s3proxy/AwsSdkTest.java]: Sorry I misread this -- you are correct.

**Pull request merged**